aac.py

```diff
@@ -8,247 +8,244 @@
-	return num_pages
+        return int(num_pages.group(1))
 
 
+    '''
+    store_animal_in_dict
 
-def scrape_page(animal_type, page_index):
-	base_url = 'http://petharbor.com/results.asp?searchtype=ADOPT&view=sysadm.v_austin&shelterlist=%27ASTN%27&where='
-	# WHERE parameter
-	if animal_type == "dog":
-		where_parameters = "type_DOG"
-	elif animal_type == "cat":
-		where_parameters = "type_CAT"
-	else:
-		where_parameters = "type_OTHER"
+    Takes 5 inputs and adds them to a dictionary using the first input as the key and the combination of all as the value (in list form).  Adds
+    these values to the instance variable self._animal_dict[]
 
-	page_parameter = '&PAGE=' + str(page_index)
+    Inputs:
+        animal_id
+        description
+        breed
+        location
+        days_in_shelter
 
-	headers = {
-		'Host': 'petharbor.com',
-		'Connection': 'keep-alive',
-		'Upgrade-Insecure-Requests': '1',
-		'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) snap Chromium/71.0.3578.98 Chrome/71.0.3578.98 Safari/537.36',
-		'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8',
-		'Referer': 'http://www.austintexas.gov/adoptablepets',
-		'Accept-Encoding': 'gzip, deflate',
-		'Accept-Language': 'en-US,en;q=0.9'
-	}
+    Return: 
+        None
 
-	page_URL = base_url + where_parameters + page_parameter
-	#print(f"Page URL is: {page_URL}")
+    '''
+    def store_animal_in_dict(self, animal_id, description, breed, location, days_in_shelter):
+        templist = []
 
-	page = requests.get(page_URL,headers=headers)
-	print(f"Scraped page {page_index}")
-	return page
+        templist.append(animal_id)
+        templist.append(description)
+        templist.append(breed)
+        templist.append(location)
+        templist.append(days_in_shelter)
+        self._animal_dict[animal_id] = templist
 
 
+    def show_animals(self):
+        for animal_id, animal in self._animal_dict.items():
+            print(animal_id, animal)
 
-def show_animals():
-	for animal_id, animal in _animal_dict.items():
-		print(animal_id, animal)
 
+    '''
+    begin_menu
 
-def scrape_for_dogs(animal_type, page_index):
-	
-	page = scrape_page(animal_type, page_index)
-	num_pages = process_page(page)
+    Draws the options on the console.
 
-	if 0 < page_index < num_pages:
-		sleep(2)
-		scrape_for_dogs(animal_type, page_index + 1)
+    Inputs
+        None
 
+    Returns:
+        None
 
+    '''
+    def begin_menu(self):
+        print(30 * "-" + " MENU " + 30 * "-")
+        print("1. Scrape AAC Dogs")
+        print("2  View Dogs")
+        print("3  Run Tests")
+        print("10  Exit")
+        print(67 * "-")
 
-'''
-begin_menu
 
-Draws the options on the console.
+    '''
+            TESTING FUNCTIONS
+    '''
+    def tests_main(self):
+        scraper = AACScraper()
+        scraper.begin_menu()
+        user_response = input("Did the menu display on the page correctly? Y or N")
```

Comment:
We probably want to drop this test code and implement it in pytest or some other testing lib. That was we can assert whether or not the scraper parsed the correct age, etc. I think you mentioned this recently though so don't mind me if this is just leftover code.